src/player.h

```diff
@@ -183,6 +183,8 @@ class Player
 	v3f eye_offset_third;
 	v3f eye_offset_third_front;
 
+	float camera_roll = 0.0f;
```

Comment:
Can this field be private?

Reply:
I could make it private and then add getter/setter functions, but it also appears that many similar fields such as eye offset are public. I am a little unsure what makes it that certain fields should be public vs private.

Reply:
Either way, I have now made the field private in https://github.com/minetest/minetest/pull/14333/commits/c127a09bcc188d3cf4af822e0dd2ed99e2f8a446.

Reply:
The use of a class usually implies that it enforces some invariants, so its members should be private in order to prevent some other code from modifying a variable and breaking an invariant.